internal/reporter/gitlab/reporter.go

```diff
@@ -171,9 +188,52 @@ func (r *reporter) SendPullRequestTriggerResult(configCtrl internal.ConfigContro
 	return nil
 }
 
-func (r *reporter) post(gitlabConfig *s2hv1.ReporterGitlab,
-	repository, commitSHA, labelName, targetURL, description string,
-	commitStatus gitlab.CommitStatus, event internal.EventType) error {
+// SendPullRequestTestRunnerPendingResult implements the reporter SendPullRequestTestRunnerPendingResult function
+func (r *reporter) SendPullRequestTestRunnerPendingResult(configCtrl internal.ConfigController,
```

Comment:
This function is doing the thing similar to `SendPullRequestQueue`
Could we combine?

Reply:
it handles different parameter type, so i combined the duplicate step but still have separate functions